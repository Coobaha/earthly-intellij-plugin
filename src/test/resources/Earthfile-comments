# when called, currently nothing is exported due to the reported bug i'm trying to fix.
# however if I were to keep track of all WAIT/END blocks (to then call the equivalent setDoSaves on them)
# we would then have to re-run all code below the END (i.e. we need to copy yesno again, and the RUN below it)
#
# Previous to WAIT/END it was impossible to have a target that could execute differently based on if it was called via a FROM or BUILD
# but now our targets can depend on how they were invoked.
both:
    WAIT
        BUILD +indirect
    END
    WAIT
        BUILD +direct
    END