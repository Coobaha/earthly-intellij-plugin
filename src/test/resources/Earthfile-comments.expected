[COMMENT->]# when called, currently nothing is exported due to the reported bug i'm trying to fix.[LINE_BREAK->]
[COMMENT->]# however if I were to keep track of all WAIT/END blocks (to then call the equivalent setDoSaves on them)[LINE_BREAK->]
[COMMENT->]# we would then have to re-run all code below the END (i.e. we need to copy yesno again, and the RUN below it)[LINE_BREAK->]
[COMMENT->]#[LINE_BREAK->]
[COMMENT->]# Previous to WAIT/END it was impossible to have a target that could execute differently based on if it was called via a FROM or BUILD[LINE_BREAK->]
[COMMENT->]# but now our targets can depend on how they were invoked.[LINE_BREAK->]
[TARGET->]both:[LINE_BREAK->]
[WHITE_SPACE->] [WHITE_SPACE->] [WHITE_SPACE->] [WHITE_SPACE->] [COMMAND->]WAIT[LINE_BREAK->]
[WHITE_SPACE->] [WHITE_SPACE->] [WHITE_SPACE->] [WHITE_SPACE->] [WHITE_SPACE->] [WHITE_SPACE->] [WHITE_SPACE->] [WHITE_SPACE->] [COMMAND->]BUILD[WHITE_SPACE->] [TARGET_REF->]+indirect[LINE_BREAK->]
[WHITE_SPACE->] [WHITE_SPACE->] [WHITE_SPACE->] [WHITE_SPACE->] [COMMAND->]END[LINE_BREAK->]
[WHITE_SPACE->] [WHITE_SPACE->] [WHITE_SPACE->] [WHITE_SPACE->] [COMMAND->]WAIT[LINE_BREAK->]
[WHITE_SPACE->] [WHITE_SPACE->] [WHITE_SPACE->] [WHITE_SPACE->] [WHITE_SPACE->] [WHITE_SPACE->] [WHITE_SPACE->] [WHITE_SPACE->] [COMMAND->]BUILD[WHITE_SPACE->] [TARGET_REF->]+direct[LINE_BREAK->]
[WHITE_SPACE->] [WHITE_SPACE->] [WHITE_SPACE->] [WHITE_SPACE->] [COMMAND->]END